components/query_filter/utils.cc

```diff
@@ -137,3 +139,23 @@ absl::optional<GURL> ApplyQueryFilter(const GURL& original_url) {
   }
   return absl::nullopt;
 }
+
+absl::optional<GURL> ApplyPotentialQueryStringFilter(
```

Comment:
`MaybeApplyQueryStringFilter`

Reply:
also this should be in namespace `query_filter`

Reply:
done